### DISCUSSION TITLE
Π2018205/ Παραδοτέο 6

### DISCUSSION THREAD

**Discussion by despina-komo** [courses-ionio/sw#1481] [Show and tell]
Σε αυτό το παραδοτέο μας ζητήθηκε να κάνουμε μία άσκηση γραμμής εντολών με ένα λογισμικό το οποίο θα είναι χωρίς systemd.

[My project](https://github.com/despina-komo/sw/tree/2018205/projects/2018205)

**Comment by theoharissiwzos** [comment]
με συγχωρεις συναδελφε ια ηθελα να σου επισημάνω πως αυτην την εβδομάδα σύμφωνα πάντα με την [ανακοίνωση για την δραστηριότητα της 6ης εβδομάδας](https://github.com/courses-ionio/sw/discussions/1449) χρειάζεται η υλοποίηση ενός static blog. Επίσης θα ήθελα να σου αναφέρω πως στον πίνακα περιεχομένων σου  στην 6η εβδομάδα έχει μπει ανάποδα ο σύνδεσμος του discussion με της αυτοαξιολόγησης. Ελπίζω να βοήθησα
![δα](https://user-images.githubusercontent.com/99976073/227651051-f9a64790-26cd-43b0-9f9f-ee3da590b0f2.png)
